ops/storage.py

```diff
@@ -252,7 +252,7 @@ def notices(self, event_path: str):
         """
         notice_list = self._load_notice_list()
         for row in notice_list:
-            if row[0] != event_path:
+            if event_path and row[0] != event_path:
```

Comment:
This also does weird things for `event_path=""`
Should this be
```
if event_path is not None and ...
```

Reply:
that was my first implementation, but the docstring does say `If not supplied (or None/'')`.

Happy to change it to just be None if that's OK. We obviously have no tests either way :-)

Reply:
Checking for the boolness of event_path is what the SQLite storage was doing, so I think it's better to leave it like that.

Reply:
ah, for sure if I changed this one I'd have to change the other one (and their docstrings)